From 749accec83f1b300fc14b5e31fd11c372ca03927 Mon Sep 17 00:00:00 2001
From: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
Date: Sun, 17 Sep 2017 00:31:15 +0300
Subject: [PATCH] meta-linaro: python-wand: fix BB_NO_NETWORK build

This fixes python-wand tag in BSP2.23 to build with BB_NO_NETWORK=1

Signed-off-by: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
---
 meta-optee/recipes-devtools/python/python-wand_0.4.3.bb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta-optee/recipes-devtools/python/python-wand_0.4.3.bb b/meta-optee/recipes-devtools/python/python-wand_0.4.3.bb
index cc91892..41a6b8f 100644
--- a/meta-optee/recipes-devtools/python/python-wand_0.4.3.bb
+++ b/meta-optee/recipes-devtools/python/python-wand_0.4.3.bb
@@ -6,7 +6,7 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=170eafd687d4a2b950819cd5e067e6d5"
 
 SRCNAME = "wand"
 
-SRC_URI = "git://github.com/dahlia/wand.git;tag=${PV}"
+SRC_URI = "git://github.com/dahlia/wand.git;tag=81b2ce5ac117cd69e4894780925fab12ff0383b9"
 S = "${WORKDIR}/git"
 
 inherit setuptools
-- 
1.9.1

